ISPN-15069 CVE-2023-4586 Hot Rod TLS Hostname validation (#11148)
infinispan/infinispan: documentation/src/main/asciidoc/topics/proc_configuring_hotrod_encryption.adoc

To verify certificates issued to {brandname} Server, Hot Rod clients require either the full certificate chain or a partial chain that starts with the Root CA.
You provide server certificates to Hot Rod clients as trust stores.

> By default, Hot Rod clients will also perform hostname validation by matching the `dNSName` and/or `iPAddress` contained by the server certificate's `subjectAltName` extension with the expected hostname.
> 

Reviewer: ```suggestion
By default, Hot Rod clients also perform hostname validation by matching the `dNSName` or `iPAddress` or both types contained by the server certificate's `subjectAltName` extension with the expected hostname.

```
